bin/registerServerTopology.php

```diff
@@ -42,28 +42,28 @@
 $helpMessage = <<<'EOD'
 
 Global Options:
-  -u <mandatory>              username of your centreon-web account
-  -t <mandatory>              the server type you want to register:
+  -u <mandatory>              username of your centreon-web account on the TARGET NODE.
+  -h <mandatory>              URL of the TARGET NODE
+  -t <mandatory>              the server type you want to register (CURRENT NODE):
             - Poller
             - Remote
             - MAP
             - MBI
-  -h <mandatory>              URL of the Central / Remote Server target platform
-  -n <mandatory>              name of your registered server
+  -n <mandatory>              name of your CURRENT NODE
 
   --help <optional>           get information about the parameters available
-  --root <optional>           your Centreon root path (by default "centreon")
-  --fqdn <optional>           provide your server FQDN instead of IP. The FQDN must be resolvable on the Central/Remote.
+  --root <optional>           your Centreon root path on TARGET NODE (by default "centreon")
+  --fqdn <optional>           provide your FQDN or IP of the CURRENT NODE. The FQDN must be resolvable on the PARENT NODE
```

Comment:
--fqdn  provide your FQDN or IP  ===> then it should not be named as "fqdn" option :) 

--node-address ? or something more generic ? as it could be FQDN or IP